stablehlo/conversions/linalg/transforms/StablehloToLinalgReduce.cpp

```diff
@@ -348,17 +347,17 @@ struct ReduceWindowOpOnTensorsGenericConversion final
 
     llvm::SmallVector<int64_t> baseDilations;
     if (op.getBaseDilations()) {
-      baseDilations = extract1DVector(*op.getBaseDilations());
+      baseDilations = llvm::SmallVector<int64_t>(*op.getBaseDilations());
```

Comment:
Is the `llvm::SmallVector<int64_t>` needed here? Isn't that the return type as it is?

If not can we use `llvm::to_vector`?

Reply:
Same for other places in this file